### ISSUE TITLE
[Shortcopy] Disconnected full-screen modal (displays after plugin activation)

### ISSUE THREAD

**Issue by folletto** [Automattic/jetpack#11537]
Refocusing Jetpack around Security and Performance.

Update to reflect new homepage designs.
Design TBD

DEVELOPMENT LOCATION DETAILS:
- This page is located on `plugins.php` immediately after jetpack activation
- And will replace the page located on `admin.php?page=jetpack#/` as well. But please make sure they have different tracks events. 

Reach out to @jeffgolenski for questions

**Comment by jeffgolenski** [comment]
Much like the reframing I did with the connection banner in this issue, here's my idea for the new full page connection screen. Again, would be super quick to implement. See comments here: https://github.com/Automattic/jetpack/issues/11532

Notes: 
Final content: tbd
Final illo color: tbd

![Full-screen-connection-7-2-cupcake](https://user-images.githubusercontent.com/214813/54164058-a000bd00-4431-11e9-8c95-e449df8bff91.jpg)

**Comment by keoshi** [comment]
I'm loving this new direction! If Security and Performance are on the same plane in terms of offering, could we make the visuals reflect that? Quick idea of having two columns (ignore everything else, especially the illo colors):

![image](https://user-images.githubusercontent.com/390760/54195490-b2710a00-44b6-11e9-81c8-ddb8e53c239e.png)

**Comment by jeffgolenski** [comment]
@keoshi omg yeah. I love that.

**Comment by shaunandrews** [comment]
I dig the overall direction here. I wonder about the use of the primary green in the illustration (I know it says sample) as it takes away the focus from the buttons.

I'm also unsure about the repetition with the TOS line and the double primary buttons. I understand the want to have the primary action button "above the fold" but I think it makes more sense below the two feature callouts. If you reduce the size of the illustrations and shorten the descriptions you can remove the button and TOS line at the top, along with the "Keep your site secure..." heading at the bottom.

Here's a quick cut-and-paste mockup:

<img width="805" alt="image" src="https://user-images.githubusercontent.com/191598/54206215-425e8600-44ae-11e9-9cad-6128ae011a7b.png">

**Comment by jeffgolenski** [comment]
@shaunandrews Brilliant. I love that take. So much simpler and focused. The addition of "not now, thank you." makes it a lot friendlier as well. Thank you for helping along with this. I think the only changes we'd have to make is to place the TOS messaging above the CTAs for legal reasons discussed in the past (conveniently I can't find that conversation right now.

I'm going to flesh this out a little more and I think we'll be good to go.

**Comment by shaunandrews** [comment]
> The addition of "not now, thank you." makes it a lot friendlier as well.

FWIW, I just cut-and-paste from @keoshi's image above. I didn't add, only removed.

**Comment by jeffgolenski** [comment]
Just a revision. Want to get this posted before visual status - still need to update the illustration color. @shaunandrews @keoshi 

![7 2 cupcake - dark - revised](https://user-images.githubusercontent.com/214813/54290279-13651480-4581-11e9-9429-5d0b45aadfc5.jpg)

**Comment by jeffgolenski** [comment]
Per the conversation about our [connection banner, here](https://github.com/Automattic/jetpack/issues/11532#issuecomment-472648412)

Here's the updated full-screen connection page using our current color scheme, just revised content and structure. 

![7 2 cupcake - light 2 - revised](https://user-images.githubusercontent.com/214813/54322329-2d2b4980-45cb-11e9-84a2-123c8a8ed2c7.jpg)

final content tbd. @keoshi @joanrho @shaunandrews

**Comment by jeffgolenski** [comment]
@dereksmart This is ready for dev. We'll work on the content in development. I updated the issue itself to describe this page's locations. Let me know if the team has any questions there. cc @keoshi

**Comment by jeffgolenski** [comment]
@Automattic/editorial Hey friends,

Our this page, we're looking at more focus just on Security and Performance. Was wondering if we could get some help making these two descriptions a little more concise:

> 
> Jetpack free protects your site against brute force attacks and unauthorized logins. 
> 
> Our premium security servives also include unlimited backups of your entire site, spam protection, malware scanning, and automated fixes.

> Activate Jetpack’s site accelerator to load pages faster, optimize your images, and serve your images and static files (like CSS and JavaScript) from our global network of WordPress.com servers. 
> 
> Speed up your site for mobile viewers and reduce bandwidth usage, which may lead to lower hosting costs.

I know this can definitely use some help. thank you so much.

**Comment by eliorivero** [comment]
@jeffgolenski I'm working on this and would be nice to have:

- font sizes (heading, body, fine print)
- the two illos

**Comment by jeffgolenski** [comment]
@eliorivero 

Thanks for jumping in. The illustrations are already on that page, so this should be quick work. It's basically just stripping out a lot of what's there and leaving the security and performance text side-by-side. For now, let's just make the font-size of the `p` 14px. (I think it's 13px now). 

![Screen Shot 2019-03-14 at 4 50 40 PM](https://user-images.githubusercontent.com/214813/54390665-76d26d80-4679-11e9-91fa-1a8c7973b5c3.png)

**Comment by eliorivero** [comment]
Thank you!

**Comment by michelleweber** [comment]
I tightened up, condensed both to single paras and rephrased a little. I also took out some details that don't really help you on this page -- you want to get people to click the button, and you don't need every detail to get that done. I think we can lose them here without creating any confusion or being misleading.

Jetpack protects you against brute force attacks and unauthorized logins. Basic protection is always free, while premium plans add unlimited backups of your whole site, spam protection, malware scanning, and automated fixes.

Activate site accelerator tools and watch your page load times and hosting costs drop -- we'll optimize your images and serve them from our own powerful global network of servers, and speed up your mobile site to reduce bandwidth usage.
